### PR TITLE
Add instructions for building without a key.

### PR DESCRIPTION
**Pull request by bonjorno7**
The important note now only says you need a key for online features, instead of the to test the game in general.
Added Google Drive link to MomentumDev.7z.
Separated the part about creating the MomentumDev folder from the part about copying files.
Made it say you don't have to copy over your momentum folder, because you delete it afterwards anyway.

### CODE REVIEW

File: _posts/2019-09-10-guide_building_the_game.md [hunk 1]
```diff
@@ -48,15 +48,17 @@ It may also be called `origin/develop`:
...
 Download [***Visual Studio 2019***](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16).
 
 Follow [this step on the Valve Dev Wiki](https://developer.valvesoftware.com/wiki/Source_SDK_2013#Unable_to_find_RegKey_for_.vcproj_files_in_solutions_.28Windows.29) to add a missing registry key to your computer.
 
 Get a Git client like [***GitHub Desktop***](https://desktop.github.com/) if you haven't already and then clone this repository.
 
-Create an extra folder somewhere called `MomentumDev` and copy all of the files from your Steam's `steamapps/common/Momentum Mod` directory into this folder.
+If you downloaded Momentum Mod from Steam, create an extra folder somewhere called `MomentumDev`.
+
+Copy all of the files **except** the `momentum` folder from your Steam's `steamapps/common/Momentum Mod` directory into this folder.
```
Comment (Gocnak) [inline _posts/2019-09-10-guide_building_the_game.md]: I wouldn't say to do this, as it makes the next line impossible to verify if it was copied correctly. It's a one-time copy anyways, saving people from copying then deleting it won't really do much other than make it confusing.

Comment (bonjorno7) [reply _posts/2019-09-10-guide_building_the_game.md]: I think it makes more sense because the MomentumDev in Google Drive doesn't include the momentum folder. I didn't want people to get confused like "Huh I can't delete this folder because I don't have it."

Comment (bonjorno7) [reply _posts/2019-09-10-guide_building_the_game.md]: I could rephrase it to "copy everything and then delete the momentum folder" or something, but I feel this is a better way to do it.

Comment (Gocnak) [reply _posts/2019-09-10-guide_building_the_game.md]: Maybe condense line 59 and 61 to one line, and remove 63 and 65, and it should be good.

**Comment by bonjorno7** [comment]
I'll have to look at that tomorrow, not convenient enough from my phone
